frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/inventory.tsx

```diff
@@ -81,15 +81,27 @@ export const getVMStatusGroups: StatusGroupMapper = (
       filterType: 'vm-status',
     },
   };
-  vms.forEach((vm: VMKind) => {
+
+  const vmStatuses = vms.map((vm: VMKind) => {
     const vmi = (vmis || []).find((instance) => getName(vm) === getName(instance));
-    const { status } = getVMStatus({ vm, vmi, pods, migrations });
+    return getVMStatus({ vm, vmi, pods, migrations }).status;
+  });
+
+  const vmisWithoutVM = (vmis || []).filter(
+    (instance) => !vms.find((vm) => getName(vm) === getName(instance)),
+  );
+  const vmiStatuses = vmisWithoutVM.map((vmi: VMIKind) => {
```

Comment:
just a nit - no need to specify `VMIKind` type and can be one-liner
```suggestion
  const vmiStatuses = vmisWithoutVM.map((vmi) => getVMStatus({ vm: undefined, vmi, pods, migrations }).status);
```

Reply:
done